benchmarks/bench_latency.py

```
@@ -0,0 +1,374 @@
+
+        header = (
+            f"  {'Context':>8} | {'Qwerky decode':>14} | {'Llama decode':>13} | {'Decode':>7}"
+            f" | {'Q TTFT':>9} | {'L TTFT':>9} | {'TTFT':>6}"
+            f" | {'Q prefill':>10} | {'L prefill':>10} | {'Prefill':>7}"
+        )
+        log.info(header)
+        log.info("  %s", "-" * len(header.strip()))
+
+        for ctx in context_lengths:
+            q = qwerky.get((ctx, bs))
+            l = llama.get((ctx, bs))
```

Ruff (0.15.6)

[error] Ambiguous variable name: `l`

(E741)